### ISSUE TITLE
Request for API access: [Your Name]

### ISSUE THREAD

**Issue by codingstark-dev** [sunnah-com/api#867]
* **Please tell us about yourself (include an email address):**
 Name: Himanshu Maurya
Email: codingstark.com@gmail.com

* **Your purpose in using this API:**
  For training the data in openai

* **API rate limits:**
  
   * **Maximum requests per second: 5**
   * **Maximum requests per day: 1k**

* **Is your use case better served by having an offline dump of hadith data or programmatic API access?**
  
Basically yes

* **What are the languages in which would you like hadith data?**
  
English 

* **What programming language will your API client be in?**
  
nodejs
